src/app/compare/compare-tree/compare-tree.component.ts

```diff
@@ -40,15 +42,19 @@ export class CompareTreeComponent {
     highlightOpenFolders: false,
     folderBehaviourOnClick: 'select',
     expandAllFolders: true,
+    determineIconClass: SimpleFileTreeUtil.conditionalCssClass,
```

Comment:
You need to give an argument to this Utility function, right?

Reply:
No the method is passed as a reference here